### PR TITLE
Create one cookie per transaction

### PR DESCRIPTION
**Pull request by luisrudge**
Using only one cookie limits us to around 9 transactions until the cookie size limit is reached. This PR changes to create one cookie per transaction instead.

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -45,6 +45,7 @@
         });
         $('#getToken').click(async () => {
           const token = await auth0.getTokenSilently();
+          console.log(auth0, token);
```
Comment (davidpatrick) [inline static/index.html]: Intentional?

File: __tests__/transaction-manager.test.ts [hunk 2]
```diff
@@ -8,61 +11,70 @@ const transaction = {
...
+      tm.create(transaction);
+      expect(tm.get(transaction.state)).toMatchObject(transaction);
+    });
+    it('`remove` removes the transaction', () => {
+      tm.create(transaction);
+      tm.remove(transaction.state);
+      expect(tm.get(transaction.state)).toBeUndefined();
+    });
+    it('`remove` saves new transactions in storage', () => {
+      const secondState = 'stateIn2';
```
Comment (davidpatrick) [inline __tests__/transaction-manager.test.ts]: `secondState` is no longer being used

File: static/index.html (hunk 1 above)
Comment (davidpatrick) [inline static/index.html]: intentional?

File: __tests__/transaction-manager.test.ts [hunk 3]
```diff
@@ -8,61 +11,70 @@ const transaction = {
...
+      tm.create(transaction);
+      tm.remove(transaction.state);
+      expect(getStorageMock().remove).toHaveBeenLastCalledWith(
```
Comment (davidpatrick) [inline __tests__/transaction-manager.test.ts]: Not sure I understand how the test description `saves new transactions in storage` matches up with the assertion ` expect(getStorageMock().remove).toHaveBeenLastCalledWith`

File: src/transaction-manager.ts [hunk 4]
```diff
@@ -17,11 +18,18 @@ interface Transactions {
...
   constructor() {
-    this.transactions = ClientStorage.get<Transactions>(COOKIE_KEY) || {};
+    this.transactions = {};
+    ClientStorage.getAllKeys()
+      .filter(k => k.startsWith(COOKIE_KEY))
+      .forEach(k => {
+        const state = k.replace(COOKIE_KEY, '');
+        this.transactions[state] = ClientStorage.get<Transaction>(k);
+      });
+    console.log(Object.keys(this.transactions).length);
```
Comment (davidpatrick) [inline src/transaction-manager.ts]: console log

File: __tests__/transaction-manager.test.ts (hunk 3 above)
Comment (luisrudge) [reply __tests__/transaction-manager.test.ts]: copy paste 😝

**Pull request merged**